x-pack/plugin/sql/qa/server/src/main/resources/datetime.csv-spec

```diff
@@ -1570,3 +1570,173 @@ SELECT CAST (CAST (birth_date AS VARCHAR) AS TIMESTAMP) a FROM test_emp WHERE YE
 ---------------
 1965-01-03T00:00:00Z
 ;
+
+
+
+// datetime ranges
+
+dateRange0
```

Comment:
Can you try and give more meaningful names to these tests?
`dateRangeTwoDateStrings` for example. The next one `dateRange_DateMathAndDate`. And for those that just change the sign (from `<` to `>`) for example, you can add a number at the end: `dateRange_Date_MathAndDate_2`.